db/seeds/support/data_migrator.rb

```diff
@@ -94,18 +105,27 @@ def role_attrs_from_function(row, person)
 
       return unless role_type && section
 
-      [
+      section_attrs = [
         {
           person_id: person.id,
           group_id: section.id,
           type: "Group::Sektion::#{role_type}"
-        },
+        }
+      ]
+
+      if role_type === 'Freimitglied'
```

Comment:
triple equals in Ruby sehe ich extrem selten... Hat mich dazu gebracht mal nachzulesen was das genau macht :D 
 https://stackoverflow.com/a/5617086
Denke an der Stelle reicht ein `==`
bzw. weil wirs ja eh auslagern fällts weg